frontend/src/static/js/main.js

```diff
@@ -105,11 +105,28 @@ window.app = new window.Vue({
     userUpdated: false,
 
     isLoading: false,
-    isCollapsed: false,
     isTabActive: true, // to force tab wrapper to load
 
     tabType: 'empty',
-    tabInfo: {},
+    tabInfo: {
+      tabAuthorship: {
+        author: '',
+        location: '',
+        maxDate: '',
+        minDate: '',
+        name: '',
+        repo: '',
+        totalCommits: ''
```

Comment:
since `repo` and `location` are objects, should the initial values be null instead?

Reply:
Don't think so, see https://github.com/reposense/RepoSense/blob/master/frontend/src/static/js/v_summary.js#L673-L680, they are actually strings.

Reply:
my bad, made a mistake there.